src/parser/core/Combatant.ts

```diff
@@ -29,7 +68,7 @@ class Combatant extends Entity {
     if (!this.owner.boss) {
       return race;
     }
-    const boss = findByBossId(this.owner.boss.id);
+    const boss: Boss | null = findByBossId(this.owner.boss.id);
```

Comment:
Setting the type here is unnecessary as it can be implied by findByBossId's return value. Omitting it will make it easier to make changes here or to findByBossId.